pkg/types/installconfig.go

```diff
@@ -15,14 +15,16 @@ import (
 	"github.com/openshift/installer/pkg/types/openstack"
 	"github.com/openshift/installer/pkg/types/ovirt"
 	"github.com/openshift/installer/pkg/types/vsphere"
+	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	// InstallConfigVersion is the version supported by this package.
 	// If you bump this, you must also update the list of convertable values in
 	// pkg/types/conversion/installconfig.go
-	InstallConfigVersion = "v1"
+	InstallConfigVersion  = "v1"
+	WorkerMachinePoolName = "worker"
```

Comment:
Does this need to be public?
```suggestion
	workerMachinePoolName = "worker"
```